lib/matplotlib/tests/test_constrainedlayout.py

```diff
@@ -678,3 +679,16 @@ def test_constrained_toggle():
         assert not fig.get_constrained_layout()
         fig.set_constrained_layout(True)
         assert fig.get_constrained_layout()
+
+
+def test_layout_leak():
+    # Make sure there aren't any cyclic references when using LayoutGrid
+    # GH #25853
+    fig = plt.figure(constrained_layout=True, figsize=(10, 10))
+    fig.add_subplot()
+    fig.draw_without_rendering()
+    plt.close("all")
```

Comment:
I guess for completeness, there should be a `del fig` here, even if it makes no difference before or after.